### ISSUE TITLE
webapp list-runtimes doesn't work after latest update of the azure cli.

### ISSUE THREAD

**Issue by dajoen** [Azure/azure-cli#12121]
## Describe the bug

**Command Name**
`az webapp list-runtimes`

**Errors:**
```
'bool' object has no attribute 'provider'
Traceback (most recent call last):
Temp\pip-install-1o1fsnhi\knack\knack\cli.py, ln 206, in invoke
azure\cli\core\commands\__init__.py, ln 608, in execute
azure\cli\core\commands\__init__.py, ln 666, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 659, in _run_job
Local\Temp\pip-install-1o1fsnhi\six\six.py, ln 703, in reraise
azure\cli\core\commands\__init__.py, ln 636, in _run_job
azure\cli\core\commands\__init__.py, ln 306, in __call__
azure-cli-core\azure\cli\core\__init__.py, ln 493, in default_command_handler
azure\cli\command_modules\appservice\custom.py, ln 704, in list_runtimes
azure\cli\command_modules\appservice\custom.py, ln 2159, in stacks
azure\cli\command_modules\appservice\custom.py, ln 2180, in _load_stacks
AttributeError: 'bool' object has no attribute 'provider'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp list-runtimes --linux`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Shell: cmd.exe

azure-cli 2.0.81

```

**Comment by dajoen** [comment]
Got the same error in Azure Cloud Shell.

**Comment by arrownj** [comment]
@dajoen , thanks for reporting, we are looking at this.

**Comment by panchagnula** [comment]
this will be fixed by 2.0.82 - in the meantime use the EdgeBuilds same as #12159
